docs/source/user-guide/installation.md

```diff
@@ -152,3 +152,7 @@ To enable columnar shuffle which supports all partitioning and basic complex typ
 ```
 --conf spark.comet.exec.shuffle.mode=jvm
 ```
+
+### Cluster mode
+Running in cluster mode it might be needed to set addtitional `--conf spark.plugins=org.apache.spark.CometPlugin` 
+to make the cluster resource managares respect Comet memory parameters. More [Memory Tuning in cluster mode](./tuning.md#memory-tuning-in-cluster-mode)
```

Comment:
It is not specified for cluster usage, I think.

Reply:
> It is not specified for cluster usage, I think.

sorry, I dont follow. Does the comment require any fix? 🤔

Reply:
Yea, I don't think `CometPlugin` is bound to cluster mode. We don't need this section probably.

Reply:
I feel we need mention it on the installation page, as this is the first page user interacts with and they interested in this topic. Perhaps we can rephrase it somehow